operators/mongodb-atlas-kubernetes/2.12.0/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -0,0 +1,866 @@
+                  "name": "kafka-cert"
+                },
+                "protocol": "SSL"
+              }
+            },
+            "name": "kafka-config",
+            "type": "Kafka"
+          }
+        },
+        {
+          "apiVersion": "atlas.mongodb.com/v1",
+          "kind": "AtlasStreamInstance",
+          "metadata": {
+            "name": "my-streaminstance-sample"
+          },
+          "spec": {
+            "clusterConfig": {
+              "provider": "AWS",
+              "region": "VIRGINIA_USA",
+              "tier": "SP30"
+            },
+            "connectionRegistry": [
+              {
+                "name": "my-streamconnection-sample"
+              }
+            ],
+            "name": "my-instance",
+            "projectRef": {
+              "name": "my-project"
+            }
+          }
+        },
+        {
+          "apiVersion": "atlas.mongodb.com/v1",
+          "kind": "AtlasTeam",
+          "metadata": {
+            "name": "atlasteam-sample"
+          },
+          "spec": {
+            "name": "my-team",
+            "usernames": [
+              "username@mongodb.com"
+            ]
+          }
+        },
+        {
+          "apiVersion": "atlas.mongodb.com/v1",
+          "kind": "AtlasThirdPartyIntegration",
+          "metadata": {
+            "labels": {
+              "app.kubernetes.io/managed-by": "kustomize",
+              "app.kubernetes.io/name": "mongodb-atlas-kubernetes"
+            },
+            "name": "atlasintegration-sample"
+          },
+          "spec": {
+            "datadog": {
+              "apiKeySecretRef": {
+                "name": "secret-ref"
+              },
+              "region": "string",
+              "sendCollectionLatencyMetrics": "string",
+              "sendDatabaseMetrics": "string"
+            },
+            "projectRef": {
+              "name": "my-atlas-project"
+            },
+            "type": "DATADOG"
+          }
+        }
+      ]
+    capabilities: Full Lifecycle
+    categories: Database
+    createdAt: "2025-12-03T08:37:51Z"
+    description: The MongoDB Atlas Kubernetes Operator enables easy management of Clusters in MongoDB Atlas
+    features.operators.openshift.io/disconnected: "false"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    operators.operatorframework.io/builder: operator-sdk-unknown
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v4
+    repository: https://github.com/mongodb/mongodb-atlas-kubernetes
+    support: support@mongodb.com
+    containerImage: quay.io/quay.io/mongodb/mongodb-atlas-kubernetes-operator-prerelease:2.12.0
```

Comment:
@josvazg I think you need to fix this image reference - looks like a copy/paste issue